### PR TITLE
feat: remove 3id doctype

### PR DESCRIPTION
**Pull request by simonovic86**
- remove `3id` doctype
- adapt tests
- set debug level to `silent` by default

### CODE REVIEW

File: packages/ceramic-doctype-tile/src/tile-doctype.ts [hunk 1]
```diff
@@ -60,11 +60,20 @@ export class TileDoctype extends Doctype {
...
      */
     static async makeGenesis(params: DocParams, context?: Context, opts: DocOpts = {}): Promise<Record<string, any>> {
-        if (!context.did || !context.did.authenticated) {
-            throw new Error('No DID authenticated')
+        const metadata = params.metadata? params.metadata : { owners: [] }
+        const is3id = metadata.tags?.includes('3id')
+        if (is3id) {
+            if (metadata.owners.length === 0) {
+                throw new Error('The owner of the 3ID needs to be specified')
+            }
```
Comment (oed) [inline packages/ceramic-doctype-tile/src/tile-doctype.ts:68]: I don't think we need any special logic for 3IDs.

File: packages/ceramic-doctype-tile/src/tile-doctype.ts [hunk 2]
```diff
@@ -60,11 +60,20 @@ export class TileDoctype extends Doctype {
...
         }
 
-        const metadata = params.metadata? params.metadata : { owners: [] }
+        if (!is3id) {
+            // check for DID and authentication
+            if (!context.did || !context.did.authenticated) {
+                throw new Error('No DID authenticated')
+            }
+        }
```
Comment (oed) [inline packages/ceramic-doctype-tile/src/tile-doctype.ts:76]: Do we really need to make this check here?

File: packages/ceramic-doctype-tile/src/tile-doctype.ts [hunk 3]
```diff
@@ -78,18 +87,23 @@ export class TileDoctype extends Doctype {
 
         const { content } = params
         const record = { doctype: DOCTYPE, data: content, header: metadata, unique }
-        return TileDoctype._signDagJWS(record, context.did, metadata.owners[0])
+        if (!is3id) {
+            return TileDoctype._signDagJWS(record, context.did, metadata.owners[0])
+        }
```
Comment (oed) [inline packages/ceramic-doctype-tile/src/tile-doctype.ts:92]: To sign the genesis record or not should be an option passed. 3ID documents should actually have a signed genesis record (in most cases).

**Review by oed — COMMENTED** [review]
I realize that there is another change needed in IDW to make all of this work properly. Will see if I can make that change quickly!

File: packages/ceramic-doctype-tile/src/tile-doctype.ts (hunk 1 above)
Comment (simonovic86) [reply packages/ceramic-doctype-tile/src/tile-doctype.ts]: agree 👍

File: packages/ceramic-doctype-tile/src/tile-doctype.ts (hunk 2 above)
Comment (simonovic86) [reply packages/ceramic-doctype-tile/src/tile-doctype.ts]: agree 👍

File: packages/ceramic-doctype-tile/src/tile-doctype.ts (hunk 3 above)
Comment (simonovic86) [reply packages/ceramic-doctype-tile/src/tile-doctype.ts]: now every 3ID genesis record is being signed

**Review by PaulLeCam — APPROVED** [review]

**Pull request merged**